mmv1/products/compute/BackendService.yaml

```diff
@@ -960,6 +1010,31 @@ properties:
               description: |
                 An optional, arbitrary JSON object with configuration data, understood
                 by a locally installed custom policy implementation.
+  - name: 'customMetrics'
```

Comment:
Does this not need maxUtilization?

Reply:
No, only customMetrics that are set on the Backend (as opposed to BackendService) need maxUtilization.